classes/php.php

```diff
@@ -41,6 +41,11 @@ public function setBinaryPath($phpPath = null)
 			}
 		}
 
+		if (basename($phpPath) === 'hhvm')
```

Comment:
`if (defined('HHVM_VERSION'))` is more reliable. I can name my binary whatever I want.